themes/bootstrap5/scss/components/record.scss

```diff
@@ -62,6 +62,7 @@
 }
 
 // Comma-separate formatting
+.record .format { display: inline-flex; }
```

Comment:
This eliminates the whitespace before the comma, which comes from the HTML simply having line breaks in between the spans.  I didn't see any negative side effects but couldn't find another backend where it was used to test -- obviously missing something.

Reply:
Generally speaking, we go through contortions to avoid having line breaks before commas in our templates because of this problem. If using `inline-flex` is a viable solution, then we might be able to make some of our templates less ugly by wrapping lines, etc. :-)